network/containerizer/bridgepolicy.go

```diff
@@ -41,28 +50,35 @@ type BridgePolicy struct {
 
 // NewBridgePolicy returns a new BridgePolicy for the input environ config
 // getter and state indirection.
-func NewBridgePolicy(cfgGetter environs.ConfigGetter) *BridgePolicy {
+func NewBridgePolicy(cfgGetter environs.ConfigGetter, st SpaceBacking) (*BridgePolicy, error) {
```

Comment:
We should probably reach some team consensus about naming these interfaces in a consistent manner. I have seen references in the code where we call these Backends, Contexts, APIs etc.